### ISSUE TITLE
Install failure

### ISSUE THREAD

**Issue by matjazpotocnik** [Toutouwai/InputfieldSelectImages#2]
Hi, Robin! Upon installing I get:

ProcessModuleInstall: File could not be downloaded (https://github.com/Toutouwai/InputfieldSelectImages/archive/master.zip) 2: file_get_contents(https://github.com/Toutouwai/InputfieldSelectImages/archive/master.zip): Failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found
(tried: curl, fopen)

On another attempt I got:

ProcessModule: Error reported by web service: That module is not currently tracked by the modules directory

I installed the module manually.

I see that 260x260 variations are created. Is there an option to specify dimensions? like 0x260 so that admin generated thumbs can be used?

**Comment by Toutouwai** [comment]
Thanks for the report.

> ProcessModuleInstall: File could not be downloaded (https://github.com/Toutouwai/InputfieldSelectImages/archive/master.zip)

This is due to the Modules Directory not taking account of GitHub changing the default branch name for all new repositories going forward. I think the Modules Directory should use the GitHub API to retrieve the actual default branch name and I've created an issue for it to get Ryan's attention: https://github.com/processwire/processwire-issues/issues/1415

As a quick fix I've manually renamed the default branch to the old "master" name so the module should be installable from the directory now.

> I see that 260x260 variations are created. Is there an option to specify dimensions? like 0x260 so that admin generated thumbs can be used?

The intention was that the thumbnails should be the same as the default admin thumb, but I forgot that the core has a fairly complicated way of working out the thumb size. In v0.1.2 I'm using `InputfieldImage::getAdminThumb` so now it should always use the same thumb as the core InputfieldImage.
